docker-compose.yml

```
@@ -15,47 +15,41 @@ services:
-      - ARCHON_AGENTS_PORT=${ARCHON_AGENTS_PORT:-8052}
+      - ARCHON_AGENTS_PORT=8052
     networks:
       - app-network
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
     volumes:
+
       - /var/run/docker.sock:/var/run/docker.sock # Docker socket for MCP container control
       - ./python/src:/app/src # Mount source code for hot reload
       - ./python/tests:/app/tests # Mount tests for UI test execution
     extra_hosts:
```

YAMLlint (1.37.1)

[error] duplication of key "extra_hosts" in mapping

(key-duplicates)